### PR TITLE
docs(contributing): specify Chromium version format for releases

### PR DESCRIPTION
**Pull request by theDavidBarton**
**Suggestion:** use a preferred Chromium version format in changelog/release note which is compatible with pptr.dev version parsing. Follow up to https://github.com/puppeteer/pptr.dev/pull/22.  
_Update:_ As the `CHANGELOG.md` is autogenerated from the Chromium roll commit-msg that is the one that requires specification.

The original problem explained in https://github.com/puppeteer/pptr.dev/issues/20#issuecomment-776271813.

> While the regex matches: "Chromium 86.0.4240.0 (r800071)" (v5.3.0)
> But it won't match: "chromium: roll Chromium to r848005" (v7.0.0)

---

**Improvement:** moving up step 5 of [_Rolling new Chromium version_](https://github.com/puppeteer/puppeteer/blob/main/CONTRIBUTING.md#rolling-new-chromium-version) to make it more visible. Follow up to https://github.com/puppeteer/puppeteer/pull/6952#issuecomment-790366531.

---

Like this pptr.dev could remain up-to-date with Chromium versions.

Maybe my suggestion is a bit verbose, in that case, feel free to rephrase it 😊

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -257,11 +257,12 @@ The following steps are needed to update the Chromium version.
...
    To do so, run `utils/check_availability.js -rb` to find the latest suitable beta Chromium revision (see `utils/check_availability.js -help` for more options).
 1. Update `src/revisions.ts` with the found revision number.
+1. Update `versions.js` with the new Chromium-to-Puppeteer version mapping.
 1. Run `npm run ensure-correct-devtools-protocol-revision`.
    If it fails, update `package.json` with the expected `devtools-protocol` version.
 1. Run `npm run tsc` and `npm install` and ensure that all tests pass. If a test fails, [bisect](#bisecting-upstream-changes) the upstream cause of the failure, and either update the test expectations accordingly (if it was an intended change) or work around the changes in Puppeteer (if it’s not desirable to change Puppeteer’s observable behavior).
-1. Update `versions.js` with the new Chromium-to-Puppeteer version mapping.
 1. Commit and push your changes and open a pull request.
+1. In `CHANGELOG.md`, name the version using the following format to make sure [pptr.dev](https://pptr.dev/) can parse it correctly: `Chromium <version> (<revision>)`, e.g. _"Chromium 90.0.4427.0 (r856583)"_ for version `v8.0.0.`.
```
Comment (mathiasbynens) [inline CONTRIBUTING.md]: The changelog is auto-generated and should not be edited by hand. This step should be removed and instead we should ensure the commit message follows the format you're proposing.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -283,6 +284,7 @@ Releasing to npm consists of the following phases:
     1. Merge the PR.
     1. Once merged, publish the release notes from `CHANGELOG.md` using [GitHub’s “draft new release tag” option](https://github.com/puppeteer/puppeteer/releases/new).
         - **NOTE**: tag names are prefixed with `'v'`, e.g. for version `1.4.0` the tag is `v1.4.0`.
+        - **NOTE**: in the case of Chromium roll, make sure to use `Chromium <version> (<revision>)` format, e.g. _"Chromium 90.0.4427.0 (r856583)"_ for version `v8.0.0.`.
```
Comment (mathiasbynens) [inline CONTRIBUTING.md]: Same here. This should not be a manual step. Instead, we should state that roll commit messages should follow this format.

File: CONTRIBUTING.md (hunk 1 above)
Comment (theDavidBarton) [reply CONTRIBUTING.md]: Ah, indeed. You even gave me a hint that it is autogenerated in https://github.com/puppeteer/pptr.dev/pull/22#discussion_r587411203. Sorry 😅 

I've simplified it based on this. 👍

File: CONTRIBUTING.md (hunk 2 above)
Comment (theDavidBarton) [reply CONTRIBUTING.md]: I've removed this line totally as it is not a relevant warning if the commit is responsible for the autogenerated content. Thanks for the notice!

**Review by mathiasbynens — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -257,11 +257,12 @@ The following steps are needed to update the Chromium version.
...
 1. Update `src/revisions.ts` with the found revision number.
+1. Update `versions.js` with the new Chromium-to-Puppeteer version mapping.
 1. Run `npm run ensure-correct-devtools-protocol-revision`.
    If it fails, update `package.json` with the expected `devtools-protocol` version.
 1. Run `npm run tsc` and `npm install` and ensure that all tests pass. If a test fails, [bisect](#bisecting-upstream-changes) the upstream cause of the failure, and either update the test expectations accordingly (if it was an intended change) or work around the changes in Puppeteer (if it’s not desirable to change Puppeteer’s observable behavior).
-1. Update `versions.js` with the new Chromium-to-Puppeteer version mapping.
-1. Commit and push your changes and open a pull request.
+1. Commit and push your changes and open a pull request. 
+   The commit message should contain the version in `Chromium <version> (<revision>)` format to ensure that [pptr.dev](https://pptr.dev/) can parse it correctly, e.g. `'feat(chromium): roll to Chromium 90.0.4427.0 (r856583)'`.
```
Comment (mathiasbynens) [inline CONTRIBUTING.md]:
```suggestion
   The commit message must contain the version in `Chromium <version> (<revision>)` format to ensure that [pptr.dev](https://pptr.dev/) can parse it correctly, e.g. `'feat(chromium): roll to Chromium 90.0.4427.0 (r856583)'`.
```

**Comment by mathiasbynens** [comment]
Thanks!

**Pull request merged**